resources/views/admin/dashboard.latte

```diff
@@ -91,21 +191,50 @@
 
     <script>
         new Vue({
-            el: '#wrapper',
+            el: "#wrapper",
             data: {
-                preview: ""
-            }
+                preview: "",
+                dates: [
+                    {
+                        date: new Date(),
+                    },
+                ],
+                customers: {$customers},
```

Comment:
Správně by se všichni "customers" měli natáhnout z templatu do Vue proměnné, ať se dá pak s polem a objekty pracovat v JS

Reply:
do tohohle netahej zadnej JS

Reply:
Ale ono to dává logický smysl, latte ti naplní JS proměnnou logickou hodnotou, a ty pak s těmi všemi hodnotami můžeš krásně pracovat na straně JS / Vuečka. Ale nevím, celou tuhle logiku Latte / Vue naprosto nedávám :D

Reply:
nejsi tu od toho abys to chapal :D